src/views/Privacy/privacyPolicy.en.mdx

```diff
@@ -0,0 +1,171 @@
+
+6. Automated individual decision-making, including profiling – Article 22 GDPR
+
+   The data subject has the right to raise an objection to processing his or her personal data, including profiling, if the Controller processes his data based on a legitimate interest, e.g. marketing of products and services, keeping statistics on the use of individual website functionalities and facilitating the use of the website, as well as satisfaction surveys. The right to raise an objection must to be relevant with special situation of Client.
+
+   If the user sends a resignation in the form of an email from receiving marketing messages regarding products or services, this will be tantamount to the user's objection to the processing of personal data, including profiling for marketing purposes.
+
+   If the resignation will be legitimate and the Controller will have no other legal basis to process personal data, the personal data will be erasing, within the scope of objection.
+
+7. The data subject has a right to lodge a complaint to The President of the Personal Data Protection Office when his or her personal data will be breaching by Controller or on the grounds of GDPR, appropriate for the place of residence of the data subject.
+8. The address for Polish Clients it is: Urząd Ochrony Danych Osobowych ul. Stawki 2, 00-193 Warszawa, email: kancelaria@uodo.gov.pl .
+9. In the purpose of executing his rights The Client may send a proper statement to The Controller’s contact address or via e mail. In the case that the data subject submits claims resulting from the indicated rights, the Controller fulfills the request or refuses to comply with it immediately, but not later than within one month after receiving it.
+
+   The Controller provides information on action taken on data subject request without undue delay and in any event within one month of receipt of the request. That period may be extended by two further months where necessary, taking into account the complexity of case and number of the requests. The Controller informs data subject about extension within one month of receipt of the request with the reasons for the delay. The information can be send by email.
+
+## Privacy Policy changes
+
+1. The Controller’s offer may expand in the future. This means The Controller will be obliged or authorized to enter those changes into the Privacy Policy.
+2. New Privacy Policy versions will occur on the website with a proper notification attached.
+3. Every change to The Privacy Policy will be binding from the day it’s being published on the website with proper notification. All changes will be properly announced and marked/featured with a one month notice from the changes implementation within The Privacy Policy.
+
+In case of further questions surrounding The hereby given Privacy Policy, please contact The Controller using his e mail: hello@kontur.io or contact address.
```

Comment:
please contact The Controller using his e mail

feels like type his -> this 🤔

Reply:
nice catch, type confirmed and fixed